src/server/api/controllers/users-controller.js

```diff
@@ -12,7 +12,7 @@ const getUsers = async () => {
 
 const getUserByUid = async (uid) => {
   const user = await knex('users')
-    .select('user_name', 'id')
+    .select('*')
```

Comment:
I would suggest not returning everything (like `created_at`, etc).
Another suggestion is to rename the variables to something more natural e.g. `user_name` to `userName`, `profile_image` to `profileImage`.

Reply:
Yes, I was thinking about it, the database naming is all snake_case and we need to use it in be camelCase

Reply:
Should I rename it here or in the front-end?

Reply:
Controllers should take snake case and convert to camelCase. That is why we have this entire layer - to prepare data in a nice way for our frontend consumers.